spec/factories/providers.rb

```diff
@@ -73,5 +73,9 @@
     trait :next_recruitment_cycle do
       recruitment_cycle { find_or_create :recruitment_cycle, :next }
     end
+
+    trait :discarded do
```

Comment:
do we actually need this?

``` ruby
let(:discarded) do
(create(:provider)).discard
end
```

Reply:
yes, to prove it *doesn't* end up in the output